### ISSUE TITLE
bug(e2e): Firebase test-server causes websocket errors

### ISSUE THREAD

**Issue by jeffbcross** [angular/angularfire2#15]
Right now the e2e test is communicating with a real Firebase.

When [this file](https://github.com/angular/angularfire2/blob/master/tools/test-server.js) is run, and then an app is opened to connect to it, many `ERR_CONNECTION_CLOSED` errors are logged to the console. 

Virtual host configuration: 

```
127.0.0.1       test.firebaseio.com
```

Ping test.firebaseio.com:

```
PING test.firebaseio.com (127.0.0.1): 56 data bytes
```

## To Reproduce

 1. Check out [this branch](https://github.com/jeffbcross/angularfire2/tree/repro-fb-server);
 2. Run `./node_modules/.bin/http-server` from the root of the project
 3. Open http://localhost:8080/test/e2e/firebase_list/
 4. Look at console

**Comment by urish** [comment]
@jeffbcross I just checked it, reproduced, and was able to fix by switching the test server address from `test.firebaseio.com` to `test.firebaseio.test`, and updating the host file accordingly.

1. Do you want me to submit a PR ?
2. Is there a way to change the hosts file on the CI? Or do we need another workaround there?

**Comment by jeffbcross** [comment]
> Do you want me to submit a PR ?

yeah

> Is there a way to change the hosts file on the CI? Or do we need another workaround there?

Yes! It's in .travis.yml

**Comment by jeffbcross** [comment]
@urish if I made the change myself, people would see the commit and say "Jeff is so smart, he figured that out by himself." I couldn't live with the guilt of stealing your credit.

**Comment by urish** [comment]
Guilt resolved.

**Comment by jeffbcross** [comment]
@urish awesome!

Closed by 4a823054dd5ae7438690fd4602fccb3eb8cf4495
